### ISSUE TITLE
TypeError: expected str, bytes or os.PathLike object, not NoneType when launching Rockstar 

### ISSUE THREAD

**Issue by wiiznokes** [lutris/lutris#5638]
### Bug description

when launching Rockstar games launcher with wine, i get an error.

### How to Reproduce

I installed the flatpak version some time ago. I played successfully GTAIV for 6 hours now.
I wanted to continue the game and now i get this error.

I think i played a little with changing the wine version, but sticked with the default version in the end. It was working before.

### Expected behavior

Launch the launcher.

### Log output

```shell
> flatpak run net.lutris.Lutris -d
2024-10-06 14:27:02,418: Command 'fluidsynth' not found on your system
INFO     2024-10-06 14:27:02,943 [application.do_command_line:479]:Starting Lutris 0.5.17
wl_surface@6: error 0: DRM Syncobj surface object already created for surface 6
ERROR at /run/build/vulkan-tools/vulkaninfo/vulkaninfo.h:1102:vkCreateWaylandSurfaceKHR failed with ERROR_OUT_OF_HOST_MEMORY
ERROR    2024-10-06 14:27:03,011 [system.read_process_output:188]:['vulkaninfo', '--summary'] command failed: Command '['vulkaninfo', '--summary']' returned non-zero exit status 1.
INFO     2024-10-06 14:27:03,011 [startup.run_all_checks:120]: (10de:2489 1458:405a nvidia) Driver 560.35.03
DEBUG    2024-10-06 14:27:03,270 [path_cache._update_missing_games:114]:Checking for missing games
DEBUG    2024-10-06 14:27:03,306 [api.check_stale_runtime_versions:52]:Modified at dim. 06 oct. 2024 13:56:20, will update after dim. 06 oct. 2024 19:56:20
DEBUG    2024-10-06 14:27:03,319 [lutriswindow.create_runtime_updater_cb:1244]:Runtime up to date
WARNING  2024-10-06 14:27:04,817 [wine.prelaunch:1014]:No valid prefix detected in /home/fedasus/SSD/Games/Lutris/rockstar_game_launcher, creating one...
INFO     2024-10-06 14:27:04,817 [wine.create_prefix:108]:Creating a win64 prefix in /home/fedasus/SSD/Games/Lutris/rockstar_game_launcher
INFO     2024-10-06 14:27:04,817 [wine.create_prefix:126]:Winepath: GE-Proton (Latest)
ERROR    2024-10-06 14:27:04,818 [jobs.target:32]:Error while completing task <bound method wine.prelaunch of <lutris.runners.wine.wine object at 0x7f018a6dc150>>: <class 'TypeError'> expected str, bytes or os.PathLike object, not NoneType
  File "/app/lib/python3.11/site-packages/lutris/util/jobs.py", line 30, in target
    result = self.function(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/lib/python3.11/site-packages/lutris/runners/wine.py", line 1015, in prelaunch
    create_prefix(prefix_path, wine_path=self.get_executable(), arch=self.wine_arch, runner=self)
  File "/app/lib/python3.11/site-packages/lutris/runners/commands/wine.py", line 160, in create_prefix
    command.start()
  File "/app/lib/python3.11/site-packages/lutris/monitored_command.py", line 161, in start
    self.game_process = self.execute_process(wrapper_command, env)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/lib/python3.11/site-packages/lutris/monitored_command.py", line 262, in execute_process
    return subprocess.Popen(  # pylint: disable=consider-using-with
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.11/subprocess.py", line 1885, in _execute_child
    self.pid = _fork_exec(
               ^^^^^^^^^^^
ERROR    2024-10-06 14:27:04,829 [exception_backstops.wrapper:36]:Rockstar Games Launcher (wine) has encountered an error: expected str, bytes or os.PathLike object, not NoneType
Traceback (most recent call last):
  File "/app/lib/python3.11/site-packages/lutris/exception_backstops.py", line 31, in wrapper
    result = function(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/lib/python3.11/site-packages/lutris/game.py", line 761, in configure_game
    raise error
  File "/app/lib/python3.11/site-packages/lutris/util/jobs.py", line 30, in target
    result = self.function(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/lib/python3.11/site-packages/lutris/runners/wine.py", line 1015, in prelaunch
    create_prefix(prefix_path, wine_path=self.get_executable(), arch=self.wine_arch, runner=self)
  File "/app/lib/python3.11/site-packages/lutris/runners/commands/wine.py", line 160, in create_prefix
    command.start()
  File "/app/lib/python3.11/site-packages/lutris/monitored_command.py", line 161, in start
    self.game_process = self.execute_process(wrapper_command, env)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/lib/python3.11/site-packages/lutris/monitored_command.py", line 262, in execute_process
    return subprocess.Popen(  # pylint: disable=consider-using-with
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.11/subprocess.py", line 1885, in _execute_child
    self.pid = _fork_exec(
               ^^^^^^^^^^^
TypeError: expected str, bytes or os.PathLike object, not NoneType
DEBUG    2024-10-06 14:27:04,831 [game.stop_game:891]:Rockstar Games Launcher (wine) has run for 0 seconds
WARNING  2024-10-06 14:27:04,831 [game.stop_game:893]:The game has run for a very short time, did it crash?
DEBUG    2024-10-06 14:27:04,831 [application.on_game_stopped:794]:Removing 1 from running IDs
DEBUG    2024-10-06 14:27:04,846 [game.stop_game:903]:Playtime: 6 hours 12 minutes
```

### System Information

```shell
Vulkan support: YES
Esync support: YES
Fsync support: YES
Wine installed: YES
Gamescope: NO
Mangohud: NO
Gamemode: YES
Steam: YES
In Flatpak: YES
[System]
OS: GNOME 45 Flatpak runtime
Arch: x86_64
Kernel: 6.10.12-200.fc40.x86_64
Desktop: GNOME
Display Server: wayland
[CPU]
Vendor: AuthenticAMD
Model: AMD Ryzen 7 3700X 8-Core Processor
Physical cores: 8
Logical cores: 16
[Memory]
RAM: 15.5 GB
Swap: 40.4 GB
[Graphics]
Vendor: NVIDIA Corporation
OpenGL Renderer: NVIDIA GeForce RTX 3060 Ti/PCIe/SSE2
OpenGL Version: 4.6.0 NVIDIA 560.35.03
OpenGL Core: 4.6.0 NVIDIA 560.35.03
OpenGL ES: OpenGL ES 3.2 NVIDIA 560.35.03
Vulkan Version: 1.3.261
Vulkan Drivers: NVIDIA GeForce RTX 3060 Ti (1.3.280)
```

**Comment by wiiznokes** [comment]
Huh, i was my SSD which was not mounted for some reason. Probably a bag system upgrade. Fixed by mounting it using Gparted. However, i have now this error `Install umu to use Proton` but it's probably unrelated

**Comment by wiiznokes** [comment]
> However, i have now this error Install umu to use Proton but it's probably unrelated

fixed it by changing the wine version to 9.0 (system)
